Find a functionally equivalent code:android.text.format.Time.toString:COMMENT
<android.text.format.Time: String toString()>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Return the current time in YYYYMMDDTHHMMSS<tz> format
 */

@@ -1 +1,7 @@
-
+{
+    // toString() uses its own TimeCalculator rather than the shared one. Otherwise crazy stuff
+    // happens during debugging when the debugger calls toString().
+    TimeCalculator calculator = new TimeCalculator(this.timezone);
+    calculator.copyFieldsFromTime(this);
+    return calculator.toStringInternal();
+}

